### ISSUE TITLE
Testing authentication protections

### ISSUE THREAD

**Issue by Dianadec** [bcgov/cas-registration#334]

**Comment by shon-button** [comment]
Currently, Playwright e2e has been implemented for App Authentication Route which requires some configurations:

In client folder create folder `client/playwright/.auth`
Upload files into client/playwright/.auth from 1password
```
OBPS FE Auth Session cas_analyst
OBPS FE Auth Session cas_pending
```
Copy file `client/e2e/.env.local.example` to `client/e2e/.env.local`
Set `client/e2e/.env.local` values as detailed in file `client/e2e/.env.local.example`
Ensure the the example id used for "new user" in the e2e test is NOT created in the user table:
```
cd bc_obps && psql

 \c registration
registration=# DELETE FROM erc.user where user_guid='c3cd84a8-e261-4814-93e1-5bd4a7ccd638';
```
From terminal command, start the app development server:
`cd client && yarn dev `
From terminal command, run Playwright e2e tests:
`cd client && yarn e2e`
All tests should pass (24)

**Comment by BCerki** [comment]
We have some (hopefully all) endpoint tests re auth

**Comment by BCerki** [comment]
Blocked by #28

**Comment by andrea-williams** [comment]
As part of this ticket we should also be checking to ensure that external users aren't able to change the status of their operator, operation, or prime admin access request themselves by posting a request using Postman, Thunder Client, or the like (i.e., bypassing the UI). 

I haven't investigated this yet, but off the top of my head my concern is that we accept values for the "Status" field in the request body on PUT endpoints. We might need to add logic that removes any updates to the "Status" field for operators, operations, or prime admin when the request is coming from an external user.

**Comment by Dianadec** [comment]
No longer needed. Closing. CC @shon-button @andrea-williams
